src/pocketdb/repositories/ConsensusRepository.cpp

```diff
@@ -32,10 +32,7 @@ namespace PocketDb
                         -- filter registrations & deleting transactions by address
                         cross join Transactions t indexed by Transactions_Type_RegId1_RegId2_RegId3
                             on t.Type in (100, 170) and t.RegId1 = addressRegId.RowId
-                        -- filter by chain for exclude mempool
-                        cross join Chain c
-                            on c.TxId = t.RowId
-                        -- filter by Last
+                        -- filter by Last (also excludes mempool)
```

Comment:
I think that the presence of a transaction in the Chain or Last or First quite accurately indicates to us that the transaction is not in the mempool.. What do you think?

Reply:
Sure, if tx is presented in `Last`, it is guaranteed that it is also presented in `Chain`, so i remove filtering by `Chain` because we are still filtering by `Last` and result is always the same